iset.mm

```diff
@@ -64477,6 +64485,72 @@ would need to be strengthened (for example, to imply included middle) to
       DBCABDLTACDMTUKUPDULUMUPUQUNSUO $.
   $}
 
+  ${
+    $d x y z w v f A $.
+    $( Two finite intersection properties.  We show that pairwise intersections
+       of elements of a set being contained in that set implies that
+       intersections of finite subsets of that set are contained in the set.
+       The converse would appear to require an additional condition, such as
+       ` x ` and ` y ` not being equal, or ` A ` having decidable equality.
```

Comment:
> If a class is closed under pairwise intersections, then it is closed under nonempty finite intersections.

As for the converse "failing": I think it's because one should be careful in defining pairwise intersection: we probably have
```
|- ( A. x e. A A. y e. A ( x =/= y -> ( x i^i y ) e. A ) <->
      A. x ( ( x C_ A /\ x =/= (/) /\ x e. Fin ) -> |^| x e. A ) ) $=
```
intuitionistically.

Reply:
> > If a class is closed under pairwise intersections, then it is closed under nonempty finite intersections.

Ooh nice wording. I'll use this.

> As for the converse "failing": I think it's because one should be careful in defining pairwise intersection: we probably have

Exactly, I think the biconditional restated as you suggest would be provable.

We don't seem to need it yet, but perhaps some day.

Reply:
Ohh, I had missed your comment in iset.mm saying "such as x and y not being equal". Sorry !